data/Amalks920.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "Amal K S",
+	"type": "personal",
+	"bio": "Aspiring fullstack developer",
+	"links": [{
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/amal-ks-9a9567183/",
+			"icon": "FaLinkedin"
+		},
+
+		{
+			"name": "Follow me on GitHub",
+			"url": "https://github.com/Amalks920",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "Follow me on Hashnode",
+			"url": "https://hashnode.com/",
```

Comment:
The url you provided redirects to hashnode feed, you might want to redirect it to your blog page.......

Reply:
> The url you provided redirects to hashnode feed, you might want to redirect it to your blog page.......

@vedprakashnautiyal please avoid duplicating your suggestion as this causes confusion to the author of the PR